<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>실행</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStop">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>310</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>중지</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>141</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>기준봉</string>
    </property>
    <widget class="QRadioButton" name="radioButton1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>94</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1분</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>94</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3분</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>94</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>5분</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>94</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>10분</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>94</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>15분</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton30">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>94</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>30분</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton60">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>94</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>60분</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton240">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>94</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>240분</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonDay">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>94</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1일</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
